cji_smoke_test.sh

```diff
@@ -181,5 +181,19 @@ if [ "$MINIO_SUCCESS" = false ]; then
     exit 1
 fi
 
+JUNIT_TEST_OUTPUTS=(
+    "iqe-${CJI_NAME}-parallel.log"
```

Comment:
The logic here could use a bit of tweaking because the 'parallel' files may not always be present.

If this is "true": https://github.com/RedHatInsights/cicd-tools/pull/30/files#diff-10065b743d1089868450d50904d5d59c03b0a255158e1543e183b220a923bbdeR36

Then we should see both the 'parallel' and 'sequential' files in the dir. Otherwise, we should only see the 'sequential' files.

Reply:
So I noticed that even if the IQE_PARALLEL_ENABLED isn't true, we still get parallel files that have no tests run. [example](https://ci.int.devshift.net/job/RedHatInsights-compliance-frontend-pr-check/lastSuccessfulBuild/) . But i think it's a better practice to not check for those so I updated the logic to check based on that env variable.